src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -25,6 +25,7 @@ const Authenticate = (() => {
     let is_any_upload_failed     = false;
     let is_any_upload_failed_uns = false;
     let onfido_unsupported       = false;
+    let from_mt5                 = false;
```

Comment:
all of our bools should start with something similar to `is_`
```suggestion
    let is_from_mt5                 = false;
```

Reply:
it should also be moved to line 27 to be with the rest of the `is_` lets